src/target/stm32l4.c

```diff
@@ -186,11 +206,15 @@ bool stm32l4_probe(target *t)
 		return false;
 	}
 	t->driver = designator;
-	target_add_ram(t, 0x10000000, sram2_size << 10);
-	/* All L4 beside L47 alias SRAM2 after SRAM1.*/
-	uint32_t ramsize = (idcode == ID_STM32L47)?
-		sram1_size : (sram1_size + sram2_size + sram3_size);
-	target_add_ram(t, 0x20000000, ramsize << 10);
+	if (is_stm32g0)
+		target_add_ram(t, 0x20000000, sram1_size << 10);
```

Comment:
Please add braces around this.  In general, if braces are used around one part of the if/else they should be used around the other.

Reply:
Coding style, argh. For NutOS, where I code a lot too, braces are requested even for for on-line clauses.  I remembered that BMP did not like, so now I learned about the extemption that the braces are wanted for one-liner when other clauses have it too . Using braces everywhere is clearer to me, as when the code in a one-line clause is extended, there is  no need to add the braces. I have adapted the code as requested.